### ISSUE TITLE
Clicked on Sealed Secret area and no sealed secret loaded

### ISSUE THREAD

**Issue by xcloudx87** [ricoberger/sealed-secrets-web#16]
Hi there, Could you please help to check again? I have deploy this to k8s but when i click on  Sealed Secret, no sealed secret is loaded. 

Thanks in advance!

**Comment by ricoberger** [comment]
Hi @xcloudx87, I tried to reproduce the issue without success. I used the following commands for my local dev setup:

```sh
helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
helm install sealed-secrets sealed-secrets/sealed-secrets --namespace kube-system

echo -n bar | kubectl create secret generic mysecret --dry-run=client --from-file=foo=/dev/stdin -o json >mysecret.json
kubeseal <mysecret.json >mysealedsecret.json --controller-name sealed-secrets
kubectl create -f mysealedsecret.json
kubectl get secret mysecret

kubectl create namespace sealed-secrets-web

helm upgrade --install sealed-secrets-web ricoberger/sealed-secrets-web --namespace sealed-secrets-web --set image.args={"--kubeseal-arguments=--controller-name=sealed-secrets"}

kubectl port-forward svc/sealed-secrets-web 8080:80
```

How did you installed Sealed Secrets and Sealed Secrets Web?

**Comment by xcloudx87** [comment]
Hi @ricoberger ,

Thank you for your response. My setup command is the same as yours except this one:
`helm upgrade --install sealed-secrets-web ricoberger/sealed-secrets-web --namespace sealed-secrets-web --set image.args={"--kubeseal-arguments=--controller-name=sealed-secrets"}`

`--kubeseal-arguments=--controller-name=sealed-secrets`
This parameter was missing from document. 

From document it is:
![image](https://user-images.githubusercontent.com/71874570/128963087-76cc1ffd-5ea2-4682-8be8-e23e8667a747.png)

Now it works as expected. Thank you very much for your help :D

**Comment by ricoberger** [comment]
Nice, the documentation should be fixed by #17.
